src/scripts/main.js

```diff
@@ -1 +1,10 @@
 'use strict';
+
+const imageMain = document.getElementById('largeImg');
+const galleryList = document.getElementById('thumbs');
+
+galleryList.addEventListener('click', (e) => {
```

Comment:
Avoid using one-letter variable namings. It could be `error`.
Also, it's really hard to find such type of variables in big projects